.travis.yml

```diff
@@ -139,6 +153,16 @@ script:
       make -j3;
     fi
 
+after_success:
+  - if [[ "$BUILD" != "Kodi" ]] && [[ "$DOCS" == "kodi-addon-dev-kit" ]]; then
+      if [[ $(git -C $TRAVIS_BUILD_DIR  config --get remote.origin.url | tr ':' '/' | awk -F ''.com/'|'.org/'' '{print $2}' | awk -F '/' '{print $1}') == AlwinEsch ]] &&
+         [[ $TRAVIS_BRANCH == next-addonlib-rework ]]; then
```

Comment:
With this becomes document upload done on checked repository and branch.

For test purpose only set to repo `AlwinEsch` and branch `next-addonlib-rework`!
For kodi it need change to repo `xbmc` and branch `master`!

Maybe we can also use a `[[ $TRAVIS_COMMIT == "[adons] update and upload kodi-addon-devkit docs" ]]` and force on changes the commit name to this for upload if it comes in Kodi.